otter/controller.py

```diff
@@ -174,6 +204,10 @@ def delete_group(log, trans_id, group, force):
     """
 
     def check_and_delete(_group, state):
+        if state.paused:
```

Comment:
I am sorry for miscommunication but we [thought](https://github.com/rackerlabs/otter/issues/1604#issuecomment-125747741) of allowing only force deletion of a paused group. I am not sure why but I thought it is inappropriate to normally allow deleting a paused group. On second thoughts, it should be fine to just delete the group if it doesn't have servers.